packages/turbo-repository/rust/src/lib.rs

```diff
@@ -182,6 +185,24 @@ impl Workspace {
         Ok(map)
     }
 
+    pub fn get_lockfile_contents(
+        &self,
+        changed_files: &HashSet<AnchoredSystemPathBuf>,
+        workspace_root: &AbsoluteSystemPath,
+        from_commit: Option<&str>,
+    ) -> LockfileContents {
+        let lockfile_name = self.graph.package_manager().lockfile_name();
+        changed_files
+            .get(AnchoredSystemPath::new(&lockfile_name).unwrap())
+            .and_then(|_| {
+                let git = SCM::new(workspace_root);
+                let anchored_path = workspace_root.join_component(&lockfile_name);
+                let contents = git.previous_content(from_commit, &anchored_path).unwrap();
```

Comment:
We'll need to handle this error as this can totally fail. Could probably just debug log the error and return a `LockfileContents::UnknownChange`

Reply:
good call: fixed in 6494692528b812240a487a5ff9954e76c523b58c